test/new-e2e/tests/installer/windows/suites/apm-library-dotnet-package/install_no_iis_test.go

```diff
@@ -40,3 +45,21 @@ func (s *testDotnetLibraryInstallSuiteWithoutIIS) TestInstallDotnetLibraryPackag
 	// 	NoDirExists(consts.GetStableDirFor("datadog-apm-library-dotnet"),
 	// 		"the package directory should not exist")
 }
+
+func (s *testDotnetLibraryInstallSuiteWithoutIIS) TestMSIInstallDotnetLibraryFailsWithoutIIS() {
+	version := "3.13.0-pipeline.58951229.beta.sha-af5a1fab-1"
+	s.Require().Error(s.Installer().Install(
+		installerwindows.WithMSIArg("DD_APM_INSTRUMENTATION_ENABLED=iis"),
+		// TODO: remove override once image is published in prod
+		// TODO: support DD_INSTALLER_REGISTRY_URL
+		installerwindows.WithMSIArg("SITE=datad0g.com"),
+		installerwindows.WithMSIArg(fmt.Sprintf("DD_APM_INSTRUMENTATION_LIBRARIES=dotnet:%s", version)),
+		installerwindows.WithMSILogFile("install-rollback.log"),
+		installerwindows.WithMSIDevEnvOverrides("CURRENT_AGENT"),
+	))
+	defer s.Installer().Purge()
+
```

Comment:
Is there a way to check that the rollback runs in the MSI logs for instance?  Because if we fix the [behavior causing the todo above](https://github.com/DataDog/datadog-agent/pull/35315/files#diff-c30d36cd1d8f63231112482274f1ede2e3ed30429899f2cd64e669063872d432R43-R46), this will pass even if the rollback does not run.

Reply:
hmm interesting point. We should be able to do that. I created a card for it so that we can get this merged as is for 7.65.0
https://datadoghq.atlassian.net/browse/WINA-1347